answers.js

```diff
@@ -4,12 +4,329 @@ Write a program that will generate a number between 1 and 100. Your program will
 function(randomno) {
```

Comment:
Naming: Avoid using abbreviations while naming parameters, functions and variables, unless there's a strong convention for that abbreviation.

Naming: Use camel case when naming variables with two or more words.